src/app/creator-profile-page/creator-profile-nfts/creator-profile-nfts.component.ts

```diff
@@ -155,6 +151,8 @@ export class CreatorProfileNftsComponent implements OnInit {
             res.PostHashHexToPostEntryResponse[key] = value;
           });
           this.myBids = res.NFTBidEntries.map((bidEntry) => {
+            // FIXME
+            // @ts-ignore
```

Comment:
@superzordon let's sync on that

Reply:
Sounds good - I'm generally ok if we need to keep some ts-ignores in to make it work